cmd/prometheus/main.go

```diff
@@ -375,6 +377,8 @@ func main() {
 	agentOnlyFlag(a, "storage.agent.wal-compression-type", "Compression algorithm for the agent WAL.").
 		Hidden().Default(string(wlog.CompressionSnappy)).EnumVar(&cfg.agent.WALCompressionType, string(wlog.CompressionSnappy), string(wlog.CompressionZstd))
 
+	agentOnlyFlag(a, "storage.agent.ignore-wal-replay-corruption", "Ignore WAL corruption while reading it, if true this will skip all corruption records of WAL and truncate it as normal.").Default("false").BoolVar(&cfg.agent.IgnoreWALReadingCorruption)
```

Comment:
Is there a reason why this flag uses the verb "replay" while the corresponding server flag uses the verb "reading"?